Cargo.toml

```diff
@@ -65,7 +66,7 @@ features = [
 ]
 
 [lib]
-crate-type = ["cdylib", "rlib"]
+crate-type = ["staticlib", "cdylib", "rlib"]
```

Comment:
lol. Maybe worth documenting what each is needed for.

Reply:
Yeah, fair. Let me look up some good descriptions. I'm also curious if we can compile only some of these for various modes, since we'll never need a staticlib for wasm32, for instance.